physical_property/README.md

```diff
@@ -40,9 +40,13 @@ Effective permeability (log<sub>*P*<sub>*app*</sub></sub>) was measured by PAMPA
 ## What's here
 
 - [`SAMPL7_molecule_ID_and_SMILES.csv`](SAMPL7_molecule_ID_and_SMILES.csv): A `.CSV` file containing SAMPL7 challenge molecule IDs and SMILES. SMILES were provided by the [Ballatore lab](https://pharmacy.ucsd.edu/faculty/ballatore).
+<<<<<<< HEAD
```

Comment:
Looks like you might have spare characters here from after a git merge conflict...?

Reply:
These have been removed